src/pages/CoursePage.js

```diff
@@ -449,95 +203,8 @@ const SectionsHeader = styled.div`
   margin-top: calc(2vmin + 1rem);
 `;
 
-const AddBar = styled.div`
-  padding: 0.5rem;
-  height: 6vh;
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-`;
-
-const CalendarButton = styled.div`
-  font-size: 1.1rem;
-  height: 100%;
-  width: 12rem;
-  border-radius: 0.6rem;
-  padding: 0.8rem 0.5rem;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  cursor: pointer;
-  background-color: ${grey[200]};
-  margin-right: 2rem;
-  transition: 0.1s;
-
-  :hover {
-    background-color: ${grey[300]};
-  }
-
-  & > svg {
-    margin-right: 0.65rem;
-  }
-`;
-
-const DateContainer = styled.div`
-  color: ${grey[800]};
-  margin: -0.2rem 0 1rem 1rem;
-  font-size: 1.25rem;
-`;
-
-const BoldedDate = styled.span`
-  font-weight: bold;
-`;
-
-const SectionContainer = styled.div`
-  padding: 1.8vmin 2.8vmin;
-  background-color: var(--grey100);
-  width: 84%;
-  margin-left: 8%;
-  position: relative;
-
-  & > .sectionName {
-    font-size: 1.8rem;
-    font-family: var(--condensedFont);
-    color: var(--grey800);
-    margin-bottom: 0.25rem;
-  }
-
-  & > .sectionNum {
-    font-size: 1.6rem;
-    font-family: var(--condensedFont);
-    color: var(--grey700);
-    margin: 0 0 -1rem 1rem;
-  }
-
-  & > .attributes {
-    display: flex;
-    flex-wrap: wrap;
-  }
-`;
-
 const CourseSections = styled.div``;
```

Comment:
This styled component seems to be redundant. We can probably just replace its instance with a div.